templates-v7/csharp/modelInnerEnum.mustache

```diff
@@ -0,0 +1,99 @@
+        {{^isContainer}}
```

Comment:
Anchor - Implemented templates for forward compatible enums in `modelInnerEnum.mustache`, used in `modelGeneric.mustache`